create_schema.py

```diff
@@ -8,20 +8,17 @@
 import os
 
 # Imputation file location and name
-common_dir = "R:\\BERD Results System Development 2023\\DAP_emulation\\"
-output_dir = "imputation\\imputation_qa"
+input_dir = "D:\\work\\data\\res_dev"
 
-pref = "full_responses_imputed"
-suff = "_2023-11-16_v340.csv"
+pref = "Frozen_Group_Data2021_244 (1)"
+suff = ".csv"
 
 # Output folder for all schemas
 out_dir = r"config\output_schemas"
 
 # Read the top 10 rows, inferrring the schema from csv
-mydir = common_dir + output_dir
-mypath = os.path.join(mydir, pref + suff)
+mypath = os.path.join(input_dir, pref + suff)
 df = pd.read_csv(mypath, nrows=10)
-df = df.drop("index", axis=1)
 
 # Get column names  as data types as dict of strings
 types = dict(df.dtypes)
```

Comment:
Should this script be functionalised?. It appears to be a handy tool however it is being altered for each use case and that is being pushed to the repo. This seems inconvenient. Also, the comments are targeted at a specific use case (Imputation file location and name) and have not been update here.
This, in my opinion, would be much more useful as a function, or as an independent script that utilises its own config so that it does not have to be altered for each use case.

Reply:
This script is a helper, not part of the package. It's only used when we create a schema for a new dataset, if we have a CSV file example. No changes for now, but we can think what to do with it in the future - additional repo? RDSA utils?